test/std-test.cc

```diff
@@ -16,7 +16,11 @@ TEST(std_test, path) {
             "\"foo\\\"bar.txt\"");
 
 #  ifdef _WIN32
-  EXPECT_EQ(fmt::format("{}", std::filesystem::path(L"Файл.txt")),
+  // File.txt in Russian.
+  // Do not use utf-8 encoded chars in this file.
```

Comment:
This comment is a bit misleading because we actually use UTF-8 below.

Reply:
Comment updated.